metriq_gym/cli.py

```diff
@@ -66,6 +67,20 @@ def prompt_for_job(args: argparse.Namespace, job_manager: JobManager) -> MetriqG
     return jobs[selected_index]
 
 
+def get_available_providers():
+    """Get list of available providers including local simulators.
+    
+    Returns:
+        list: List of provider names including remote providers and 'local'
+    """
+    try:
+        remote_providers = get_providers()
+        return remote_providers + ["local"]
+    except Exception:
+        # Fallback if qBraid is not available or configured
```

Comment:
Hmm, I'm not sure about whether we need or want a fallback. I think we expect that `qBraid` is a necessary dependency of metriq-gym, so any of the providers that they offer (with the exception of local, I suppose) are ones that we should assume exist and are accessible.